bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -289,8 +289,15 @@ counts_above = []
 fits = []
 tpars = []
 
-for tnum in trange:
-    stat_in_template = stat[tid == tnum]
+tsort = tid.argsort()
+tid = tid[tsort]
+stat = stat[tsort]
+
+left = np.searchsorted(tid, trange, side='left')
+right = np.searchsorted(tid, trange, side='right')
+
+for j, tnum in enumerate(trange):
+    stat_in_template = stat[left[j]:right[j]]
```

Comment:
to ensure that I understand this:
- sort by template ID
- use searchsorted to find the extrema of indices of where template ID is that one
- use the extrema indices to select the stat values

Reply:
@GarethDaviesGW Yes, that's right.